src/v/config/tests/advertised_kafka_api_test.cc

```diff
@@ -34,6 +34,28 @@ static auto const advertised_kafka_api_conf = "  advertised_kafka_api:\n"
                                               "    address: 10.48.0.2\n"
                                               "    port: 1234\n";
 
+static auto const kafka_endpoints_conf_v2
+  = "redpanda:\n"
+    "  data_directory: /var/lib/redpanda/data\n"
+    "  node_id: 1\n"
+    "  rpc_server:\n"
+    "    address: 127.0.0.1\n"
+    "    port: 33145\n"
+    "  kafka_api:\n"
+    "    - address: 192.168.1.1\n"
+    "      port: 9999\n"
+    "  seed_servers:\n"
+    "    - host: \n"
```

Comment:
i don't see the test for the multiple `name, addr, port`

Reply:
I'll add that. The existing test converts to vector, but of size one